ebos/eclwriter.hh

```diff
@@ -243,18 +250,23 @@ public:
             OPM_END_PARALLEL_TRY_CATCH("Collect to I/O rank: ",
                                        this->simulator_.vanguard().grid().comm());
         }
+        
 
         std::map<std::string, double> miscSummaryData;
         std::map<std::string, std::vector<double>> regionData;
         Inplace inplace;
+        
+        if (!isSubStep)
         {
             OPM_TIMEBLOCK(outputFipLogAndFipresvLog);
-            inplace = eclOutputModule_->outputFipLog(miscSummaryData, regionData, reportStepNum,
-                                                     isSubStep, simulator_.gridView().comm());
-            eclOutputModule_->outputFipresvLog(miscSummaryData, regionData, reportStepNum,
-                                               isSubStep, simulator_.gridView().comm());
-        }
 
+            inplace = eclOutputModule_->calc_inplace(miscSummaryData, regionData, simulator_.gridView().comm());
+            
+            if (simulator_.gridView().comm().rank() == 0){
```

Comment:
I think I'd prefer we write this check as `collectToIORank_.isIORank()` instead.